NitroxServer/Server.cs

```diff
@@ -44,6 +46,51 @@ public Server(ServerConfigReader configReader)
 
         public void Start()
         {
+            NetworkInterface[] allInterfaces = NetworkInterface.GetAllNetworkInterfaces();
+            foreach(NetworkInterface eachInterface in allInterfaces)
+            {
+                if (eachInterface.Name == "Hamachi")
+                {
+                    string hamachiAddresses = "";
+                    int count = 0; //Multiple Hamachi IP's , IPv4 and IPv6
+                    foreach(IPAddressInformation hamachiIP in eachInterface.GetIPProperties().UnicastAddresses)
```

Comment:
You could make this foreach almost a one liner with LINQ to make it easier to read this method. Something like:

```
var ips = eachInterface.GetIPProperties().UnicastAddresses
    .SelectMany(address => address)
    .Where(address => address.Address.ToString().Contains("fe80::"));
Log.Info("If using Hamachi, use one of the following IPs: " + string.Join(" or ", ips))
```

It's of the top of my head so it might not compile directly or work.